integration/unplugin/README.md

```diff
@@ -84,12 +84,12 @@ interface PluginOptions {
 }
 ```
 
-- `emitDeclaration`: Whether to generate `.d.ts` type definition files from the Civet source, which is useful for building libraries. Default: `false`
+- `emitDeclaration`: Whether to generate `.d.ts` type definition files from the Civet source, which is useful for building libraries. Default: `false` (Requires installing `typescript`)
 - `typecheck`: Whether to run type checking on the generated code. Default: `false`.
-- `outputExtension`: Output filename extension to use. Default: `.civet.tsx`, or uses `.civet.jsx` if `js` is `true`.
-- `ts`: Mode of transpilation of TS->JS source code. Default: `civet`. Options:
-  - `civet`: Use Civet's JS mode.
-  - `esbuild`: Use esbuild's transpiler. (Requires installing `esbuild`)
-  - `tsc`: Use the typescript compiler. (Requires installing `typescript`)
-  - `preserve`: Don't transpile TS code.
+- `outputExtension`: Output filename extension to use. Default: `".civet.tsx"`, or uses `".civet.jsx"` if `js` is `true`.
+- `ts`: Mode of transpilation of TS->JS source code. Default: `"civet"`. Options:
+  - `civet`: Use Civet's JS mode (Not all TS features supported)
+  - `esbuild`: Use esbuild's transpiler (Requires installing `"esbuild"`)
+  - `tsc`: Use the typescript compiler (Requires installing `"typescript"`)
+  - `preserve`: Don't transpile TS code (Some bundlers like esbuild and vite can handle TS directly. Note that this would require additional plugins for bundlers that can't handle TS)
```

Comment:
```suggestion
  - `preserve`: Don't transpile TS code (Some bundlers like esbuild and Vite can handle TS directly. Note that this would require additional plugins for bundlers that can't handle TS, such as Rollup.)
```

Mentioning Rollup which isn't obvious because there is a TypeScript plugin, but (as you know) it doesn't work with additional plugins like ours.

Also, "vite" → "Vite".

Reply:
Hm maybe we should reword this statement if we are mentioning rollup, because in case of rollup it infact _won't_ work with even an additional plugin